crowbar_framework/Gemfile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-source "https://rubygems.org"
+source "http://rubygems.org"
```

Comment:
The 3 people should be using SLE12 .. :-) https://trello.com/c/2sw8Wc0M/426-5-p9-make-crowbar-rails-app-work-on-sle12